packages/core/admin/admin/src/StrapiApp.tsx

```diff
@@ -552,46 +629,188 @@ class StrapiApp {
       },
     });
 
-    const {
-      components: { components },
-      fields: { fields },
-    } = this.library;
-
-    return (
-      <BrowserRouter basename={getBasename()}>
-        <Providers
-          components={components}
-          fields={fields}
-          customFields={this.customFields}
-          localeNames={localeNames}
-          getAdminInjectedComponents={this.getAdminInjectedComponents}
-          getPlugin={this.getPlugin}
-          messages={this.configurations.translations}
-          menu={this.menu}
-          plugins={this.plugins}
-          runHookParallel={this.runHookParallel}
-          runHookWaterfall={(name, initialValue, async = false) => {
-            return this.runHookWaterfall(name, initialValue, async, store);
-          }}
-          // @ts-expect-error – context issue. TODO: fix this.
-          runHookSeries={this.runHookSeries}
-          themes={this.configurations.themes}
-          settings={this.settings}
-          store={store}
-        >
-          <Helmet
-            htmlAttributes={{ lang: localStorage.getItem(LANGUAGE_LOCAL_STORAGE_KEY) || 'en' }}
-          />
-          <App
-            authLogo={this.configurations.authLogo}
-            menuLogo={this.configurations.menuLogo}
-            showTutorials={this.configurations.tutorials}
-            showReleaseNotification={this.configurations.notifications.releases}
-          />
-        </Providers>
-      </BrowserRouter>
+    const settingsRoutes = [...ROUTES_EE(), ...ROUTES_CE].filter(
```

Comment:
One accessing window so you want to run that function as opposed to calculate at build time – the latter would throw in Vite I think, we had similar issues with the `getBasename` call too previously.

Reply:
how about making it clearer that it's a function then? Something like getEERoutes()? since it's not a constant I don't think the screaming snake case is appropriate

Reply:
sure :)